website/content/docs/developers/simulator-mode.md

```diff
@@ -39,8 +40,22 @@ simulator.deactivate("test");
 simulator.shutdown();
 ```
 
-To kill a topology, you could also simply terminate this process.
+To kill a topology, one could also terminate the process.
+
+The simulator mode will run in separate threads other than main thread. All those interfaces are 
```

Comment:
...the main thread.

What do you mean by "All those interfaces"? It's not clear what you're referring to here? Do you mean to say "The `Simulator` class is thread safe."?